card/mod/utility/set/abstract/media.rb

```diff
@@ -21,13 +12,6 @@ def text_with_image opts={}
   private
 
   def normalized_text_with_image_opts opts
-    opts.reverse_merge! title: _render_title,
-                        text: "",
-                        src: image_src(opts),
-                        alt: image_alt,
-                        size: :original,
-                        media_opts: {},
-                        media_left_extras: "",
-                        media_left: ""
+    opts.reverse_merge! title: _render_title, text: "", size: voo.size, media_opts: {}
```

Comment:
No more `src` parameter for image tags doesn't seem right to me.

Reply:
See media_snippet.haml below. Now we're nesting the image card and configuring as necessary with `items`.  The image-box is part of the image view. 

Note that this approach has allowed a lot more standardization (eg with thumbnails), less code overall, and it made it really simple to have the new "default" images (actually icons) on wikirate.

Reply:
Also, supporting "src" is begging for caching issues.  nesting is the decko way.

Reply:
Does it still look good if there is no image? I think that it was hard work for Vignesh to achieve that.

Reply:
yes.  I'm not sure he had mastered the unknown handling (the api for which has since improved).